messages_appendix.cddl

```diff
@@ -67,12 +67,18 @@ psk-input-method = &(
   qr-code: 1
 )
 
+auth-initiation-token = (
+  ? 0 : text ; token
+)
+
 ; type key 1002
 auth-spake2-need-psk = {
+  auth-initiation-password
```

Comment:
I think here and below should be "-token" since you made it "-token" everywhere else.

Reply:
+1

Reply:
I'll fix this in a follow-up commit.